LICENSE

```diff
@@ -200,3 +200,30 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 
+----
+
+This product bundles Glyphicon Halflings ( http://glyphicons.com/
+ ) via Bootstrap 3's distributing these under The MIT License.
+
+This product bundles angular-ui / ui-sortable / sortable.js ( https://github.com/angular-ui/ui-sortable/blob/master/src/sortable.js ) under The MIT License.
+
+
+The MIT License (MIT)
```

Comment:
I don't believe references to `node_modules` or URLs are sufficient, no. If we're distributing the bits of the third party software, we have to also distribute the bits of the third party software's license.

`node_modules` aren't in the distribution as such -- they're stuff node pulls down in resolving pointers in package metadata. It's subtle, but the point is those copies of the LICENSEs aren't actually in our "distribution", in the thing we're distributing.

Likewise, a link doesn't fulfill the MIT license requirement that 

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

!L. Not legal advice.